clients/src/test/java/org/apache/kafka/clients/admin/AdminClientTestUtils.java

```diff
@@ -78,6 +80,19 @@ public static ListTopicsResult listTopicsResult(String topic) {
         return new ListTopicsResult(future);
     }
 
+    /**
+     * Helper to create a AlterConfigsResult instance for a given Throwable.
+     * AlterConfigsResult's constructor is only accessible from within the
+     * admin package.
+     */
+    public static AlterConfigsResult alterConfigsResult(ConfigResource cr, Throwable t) {
+        KafkaFutureImpl<Void> future = new KafkaFutureImpl<>();
+        Map<ConfigResource, KafkaFuture<Void>> futures = new HashMap<>();
+        futures.put(cr, future);
```

Comment:
We can use singletonMap here:
```suggestion
Map<ConfigResource, KafkaFuture<Void>> futures = Collections.singletonMap(cr, future);
```